packages/pods-core/src/builtin/NotionPod.ts

```diff
@@ -0,0 +1,156 @@
+import { DendronError, ERROR_SEVERITY, NoteProps } from "@dendronhq/common-all";
+import { markdownToBlocks } from "@instantish/martian";
+import type {
+  Page,
+  TitlePropertyValue,
+} from "@notionhq/client/build/src/api-types";
+import { Client } from "@notionhq/client";
+import _ from "lodash";
+import { ExportPod, ExportPodPlantOpts, ExportPodConfig } from "../basev3";
+import { JSONSchemaType } from "ajv";
+import { NotionUtilMethods, PodUtils } from "../utils";
+import { RateLimiter } from "limiter";
+
+const ID = "dendron.notion";
+
+// Allow 3 req/sec. Also understands 'hour', 'minute', 'day', or a no. of ms
```

Comment:
nit: can you specify in the comment that this 3 req/sec comes as a limitation on the API from the Notion side?